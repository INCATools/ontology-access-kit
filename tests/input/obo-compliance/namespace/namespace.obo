format-version: 1.4
ontology: namespace

[Term]
id: X:1
namespace: NS1

