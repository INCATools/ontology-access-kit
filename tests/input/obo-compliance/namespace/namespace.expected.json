{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/namespace.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBONamespace",
          "val" : "NS1"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBONamespace",
      "lbl" : "has_obo_namespace",
      "type" : "PROPERTY"
    } ]
  } ]
}